service/service.go

```diff
@@ -525,6 +525,19 @@ func configureViews(level configtelemetry.Level) []config.View {
 		}))
 	}
 
+	// Batch exporter metrics
+	scope = ptr("go.opentelemetry.io/collector/exporter/exporterhelper")
+	if level < configtelemetry.LevelNormal {
+		views = append(views, dropViewOption(&config.ViewSelector{
+			MeterName: scope,
+		}))
+	} else if level < configtelemetry.LevelDetailed {
```

Comment:
Unlike the batchprocessor, I think the exporterhelper would be considered part of "the core Collector API", so metrics other than the new one should stay available at `level: basic:` like they are now.
```suggestion
	if level < configtelemetry.LevelDetailed {
```